lib/openstack/openstack_handle/handle.rb

```diff
@@ -25,16 +25,19 @@ class Handle
       "Orchestration" => :orchestration
     }
 
+    def self.try_connection_with_ssl
```

Comment:
So, this is really try_first_good_http*_connection, right?  I'd probably drop the `_with_ssl` from the name since we're not really preferring ssl.  If http is listening and responds without error, we'll use that, right?

I don't know what to recommend for a name because the underlying problem is still there.